config/Makefile.am

```diff
@@ -241,6 +241,7 @@ CONFIG_FILES = \
 	services/spideroak-lansync.xml \
 	services/squid.xml \
 	services/ssh.xml \
+	services/steam.xml \
```

Comment:
I think "steam-streaming" is a better name as there are other ports used by steam for file downloads, etc.